resources/prosody-plugins/mod_speakerstats_component.lua

```diff
@@ -6,6 +6,13 @@ local st = require "util.stanza";
 local socket = require "socket";
 local json = require "util.json";
 
+-- we use async to detect Prosody 0.10 and earlier
```

Comment:
Can we just drop support for those?

Reply:
Not really if we do not want to drop support for default deployment on 18.04 LTS.

Reply:
Ack